pkg/webhook/admission/builder/builder.go

```diff
@@ -159,6 +171,23 @@ func (b *WebhookBuilder) validate() error {
 	return nil
 }
 
+func (b *WebhookBuilder) setDefaults(w *admission.Webhook) {
+	if len(b.path) == 0 {
+		if *b.t == types.WebhookTypeMutating {
+			b.path = "/mutate-" + w.Rules[0].Resources[0]
```

Comment:
do resources have a group in them?  Will this end up as `resource.group` (meaning we need to do substitution?)

Reply:
> do resources have a group in them?

No. Group is in `w.Rules[0].Groups`. Rule has the following struct:
```
type Rule struct {
	APIGroups []string
	APIVersions []string
	Resources []string
}
```
We are not using `group` name in the path to keep it short.